docs/ACCESSIBILITY.md

```diff
@@ -48,9 +50,81 @@ We welcome your feedback and will continue to improve the product in this area,
 
 -  A [focusable](#focusable) element which can be focused on by pressing <kbd>TAB</kbd> key
 
+## UX: Navigating activities in the transcript
+
+> This is related to [#2996](https://github.com/microsoft/BotFramework-WebChat/issues/2996).
+
+### User story
+
+The user should be able to navigate across multiple activities in the transcript using navigation keys, such as <kbd>UP</kbd>, <kbd>DOWN</kbd>, <kbd>HOME</kbd> and <kbd>END</kbd> keys. The navigation keys are based on the [WAI-ARIA best practices for grid widget](https://www.w3.org/TR/wai-aria-practices-1.1/#grid).
+
+Optionally, when the user press <kbd>ESCAPE</kbd>, it should blur the focus on the transcript and send the focus to the send box.
+
+Although transcript can be navigated using navigational keys and keyboard visual indicator is placed around the activity and the transcript, the navigation is not considered a selection (such as `aria-selected="true"`).
+
+### Focus redirector
+
+Focus redirector is an element to capture and redirect focus, enabling <kbd>TAB</kbd> to jump in a specific sequence that would otherwise, impossible to jump using tab sequence.
+
+The element itself is a focusable and invisible. When focused, it will programmatically send the focus to a target element.
+
+They are commonly used in modal dialog as focus trap, a mechanism to prevent focus to move outside of the dialog.
+
+Note that, on focus, browser will scroll element into view. This behavior is not preventable (tested through `event.preventDefault()` on `focus` event listener in bubbling and capturing phase). Thus, when placing focus redirector inside a scrollable container, some considerations should be made about its size and position.
+
+### Hiding focusable elements in the transcript
+
+While focusing on the transcript, when the user press <kbd>TAB</kbd>, it should skip all focusables in the transcript and focus on the next tabbable element after the transcript.
+
+To avoid any DOM elements under a subtree to be focusable, HTML introduced a new [`inert` attribute](https://html.spec.whatwg.org/multipage/interaction.html#inert-subtrees). However, this attribute is not widely adopted. Although polyfill is available, it will possibly introduce performance penalty. Thus, we are not using `inert` attribute.
```

Comment:
```suggestion
To avoid any DOM elements under a subtree being focusable, HTML introduced a new [`inert` attribute](https://html.spec.whatwg.org/multipage/interaction.html#inert-subtrees). However, this attribute is not widely adopted. Although polyfill is available, using it may introduce a performance penalty. Thus, we are not using `inert` attribute at this time.
```